semantics.json

```
@@ -61,10 +61,10 @@
         }
       },
       {
-        "name": "coverMedium",
+        "name": "coverMedia",
```

We don't want make a change like this unless there is a good reason for it. Changing the actual field name will require code changes and an upgrade script, and it adds zero value for users. Changing the label and description is all that is needed.

```suggestion
        "name": "coverMedium",
```